assets/components/odf-lvm/topolvm-openshift-storage_namespace.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: openshift-storage
+  annotations:
+    openshift.io/node-selector: ""
+    workload.openshift.io/allowed: "management"
+  labels:
+    # ODF-LVM should not attempt to manage openshift or kube infra namespaces
+    topolvm.cybozu.com/webhook: "ignore"
```

Comment:
I don't understand what this is doing. Why do we need so many things to have this label and why shouldn't openshift or kube-infra namespaces be seen by TopoLVM?

Reply:
Actually, I believe this is only a requirement when running the extended scheduler.  Will remove.